doc/migrations/3.3.0_3.3.0plus.md

```diff
@@ -0,0 +1,58 @@
+## New field in Message Archive Management MUC entries: Sender ID
+
+As a part of ensuring GDPR compliance, it turned out it's essential to be able to efficiently query MAM MUC data via sender ID (to retrieve user's personal data).
+Originally, the sender JID could be found only as a part of encoded XML message element, so finding all items sent by a certain user would be **extremely** inefficient (or rather: anti-efficient).
```

Comment:
Originally, the sender JID could be found only as a part of an encoded XML message element, so finding all items sent by a certain user would be **extremely** inefficient (or rather: anti-efficient).